conf/messages

```diff
@@ -156,6 +156,7 @@ nml.edge.invalid=Invalid edge {0} {1}
 nml.tree.id.invalid=Invalid tree id {0}
 nml.tree.elements.invalid=Invalid {0} in tree {1}
 nml.treegroup.id.invalid=Invalid tree group id {0}
+nml.boundingbox.id.invalid=Invalid user bounding box id {0}
```

Comment:
@youri-k Does this line mean that an id of `0` for a user bounding box is invalid and will result in an error? Currently the frontend gives the first bounding box of each tracing the id of 0 ...

Is there any specific reason why id 0 shall result in an error?

Reply:
No, this is string templating for error messages, so we'll supply an addiotional String parameter and the MessageProvider in the backend generates a human readable error message. In my mind, the id 0 should be reserved for the original user bounding box, but if there is none, the 0 is a valid id.